app/lib/actions/katello/repository_set/available_repositories.rb

```diff
@@ -0,0 +1,51 @@
+module Actions
+  module Katello
+    module RepositorySet
+      class AvailableRepositories < Actions::AbstractAsyncTask
+        def plan(product:, content_id:)
+          results = []
+          cdn_config = product.organization.cdn_configuration
+
+          if ::Katello::Resources::CDN::CdnResource.redhat_cdn?(cdn_config.url)
```

Comment:
One consequence of this change is that it will assume anything that isn't 'cdn.redhat.com' is another katello instance.  So something like cdn.stage.redhat.com or a manually configured http:// cdn would not work i presume (although stage is maybe the better use case).   I guess the user could manually set the SETTING to their url, but i'm not sure the installer supports that.   Maybe having a cdn type field or other configuration would help make this clear cut?